src/domain/app/App.tsx

```diff
@@ -91,6 +92,14 @@ const App: React.FunctionComponent = props => {
           <ProfileRoute />
         </PrivateRoute>
 
+        {
+          // TODO: The route should probably be /profile/child/${child.id}/events/${event.id}
```

Comment:
This routing is ridiculous, i think /event/eventid will be neat

Reply:
Fair enough!